webservice/bagitutils.py

```diff
@@ -212,9 +213,13 @@ def participants_and_max_files_in_sample_and_protocols(self):
                 if protocol is not None:
                     native_protocols.add(protocol)
 
-        return participants, max(samples.values()), native_protocols
+            # Track all file types in manifest.
+            for file_type_ in row[BoardwalkColumns.FILE_TYPE].split(','):
```

Comment:
Why the `split()`? I downloaded the manifest from dev, and I don't see any commas in that column.

I may be missing something, or maybe the input is different for the export case, but this seems unnecessary.

Reply:
I did some testing. Both lists, `row[BoardwalkColumns.FILE_URLS]` and `row[BoardwalkColumns.FILE_TYPE]`, I tested only had one item, and this may always be the case. We could take it out based on that. That said, I would suggest erring on the safe side and leave on the off-chance that for some reason that there's more than 1 item in that list. After all the split is silent. What do you think?

Reply:
You still have the `split`.

Reply:
To clarify this point: `BoardwalkColumns.FILE_TYPE` contains a comma-separated list of all file types in the manifest. Before iterating over it in the for-loop we need the split. So AFAIK it's not up for vote...

Reply:
Can you please show an example where there is a comma-separated list. This is a column, for each row, the value is a single file type.